### ISSUE TITLE
Bug: Turn & publish 

### ISSUE THREAD

**Issue by louparis** [quadproduction/OpenPype#549]
### Before posting a new ticket, have you looked through the documentation to find an answer?

Yes I have

### Have you looked through the existing tickets to find any related issues ?

Yes I have

### Author of the bug

Loïc Bail

### Version

3.15.12-quad.4.1

### What platform you are running OpenPype on?

Windows

### Current Behavior:

soucis quand je lance la turn et le publish de cet asset dans le pipe shotgrid :

https://wizz.shotgunstudio.com/page/217325#Asset_56891

### Expected Behavior:

Quand je suis dans maya, menu shotgrid, submit to muster, render, je devrais normalement avoir trois jobs dans la queue list de muster. un pour le calcul des images, puis un pour le calcul de la review, et un dernier pour le publish
a l'heure actuelle, je n'ai que le calcule des images de la turn. Pas de review, pas de publish.

### What type of bug is it ?

Happened only once in a particular configuration

### Which project / workfile / asset / ...

cf lien

### Steps To Reproduce:

Demander à Loïc si besoin de plus d'infos

**Comment by mre7a** [comment]
I tested and it works fine !

![image](https://github.com/quadproduction/OpenPype/assets/68907585/65bcd64b-4934-4820-bd4e-483bb13c3ee6)

![image](https://github.com/quadproduction/OpenPype/assets/68907585/0ac9da16-b978-4158-a445-561f353cbe9c)

**Comment by mre7a** [comment]
They use underscore in the name of their assets, which is not accepted by shotgrid pipe. 
They clean up their assets and see if the problem persists.
